src/assets/translations/es-MX.json

```diff
@@ -923,7 +921,7 @@
       "External Asset Owner": "Propietario de activos externos",
       "External Services": "Servicios externos",
       "Family Member": "Miembro de la familia",
-      "Family Members": "Referencias Personales y Familiares",
+      "Family Members": "Miembros de la familia",
```

Comment:
Please keep this translation for Spanish ""Referencias Personales y Familiares""